Joker/src/types.ts

```diff
@@ -0,0 +1,9 @@
+export type RootStackParamList = {
+  home: undefined;
```

Comment:
Look [typescript guide](https://reactnavigation.org/docs/typescript#type-checking-screens-and-params-in-nested-navigator) in react-navigation for nested navigators